### ISSUE TITLE
kafka scala version for metrics reader is hardcoded to 2.11

### ISSUE THREAD

**Issue by manuzhang** [intel-hadoop/HiBench#390]
The Scala version is 2.10 for Spark Streaming 1.6, which will cause runtime error since both versions are loaded

**Comment by carsonwang** [comment]
Yes! Another issue is that Spark 1.6 is hardcoded to Scala 2.10 while Spark 2.0 is hardcoded to Scala 2.11.
We may need add a profile to allow user specifying the Scala version when building HiBench.

**Comment by zybing** [comment]
@manuzhang I fixed this problem by specify scala version when build hibench. such as : mvn clean package -Dscala=2.10 . So can you try my pr(#399) to verify whether your problem has been solved.
